### DISCUSSION TITLE
Welcome to cleopatra Discussions!

### DISCUSSION THREAD

**Discussion by moesaid** [moesaid/cleopatra#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by bagusseno** [comment]
Sorry but, is the input not styled?
